docs/encyclopedia/workers.mdx

```diff
@@ -190,6 +191,30 @@ Once an Activity Task finishes execution, the Worker responds to the Temporal Se
 - ActivityTaskTerminated
 - ActivityTaskTimedOut
 
+## What is a Nexus Operation Task? {#nexus-operation-task}
+
+A Nexus Operation Task provides the context needed to proceed with an Nexus Operation Task Execution.
+Nexus Operation Tasks largely represent the Nexus Operation Scheduled Event, containing the data needed to execute a Nexus Operation handler.
+
+## What is a Nexus Operation Task Execution? {#nexus-operation-task-execution}
+
+An Nexus Operation Task Execution occurs when a Worker uses the context provided from the Nexus Operation Task and executes the Nexus Operation handler (also known as the Nexus Operation handler function).
+
+The NexusOperationScheduled Event corresponds to when the Temporal Service puts the Nexus Operation Task into the Task Queue.
+
+The NexusOperationStarted Event corresponds to when the Worker picks up the Nexus Operation Task from the Task Queue.
+
+Either NexusOperationCompleted or one of the other Closed Nexus Operation Events corresponds to when the Worker has yielded back to the Temporal Service.
+
+The API to schedule a Nexus Operation Execution provides an "effectively once" experience, even though there may be several Nexus Operation Task Executions that take place to successfully complete an Nexus Operation.
```

Comment:
```suggestion
The API to schedule a Nexus Operation Execution may provide an "effectively once" experience, depending on the implementation of the Nexus Operation handler (which should be idempotent), even though there may be several Nexus Task Executions that take place to successfully complete an Nexus Operation, for example in the case of retries.
```

Reply:
How about this?

A Nexus Operation Execution appears to the caller Workflow as a single RPC, while under the hood the Temporal Service may issue several Nexus Tasks to attempt to start the Operation. Hence, a Nexus Operation Handler implementation should be idempotent. The WorkflowRunOperation provided by the SDK leverages Workflow ID based deduplication to ensures  idempotency and provide an "effectively once" experience.